### PR TITLE
Simplify and getStatistics

### PR DESCRIPTION
**Pull request by shehzan10**
Based on @pjcozzi suggestion, this PR does 2 things:

* Removed `PrimitiveHelpers.js` and `isDataUri.js` as dependencies of `getStatistics`
  * This require duplicating a few lines of code.
* Adds a `gulp` command `build-cesium-simplify` that exports the listed files into a independent class format that only depends on Cesium already being imported (no require).
  * The files eligible for this must only depend on Cesium and no other files.
  * This can probably be expanded in the future to require local files as well.

Note: The nomenclature (`simple`/`simplify`) maybe naive. So if there is a technical term for it, then I can change it to that.

### CODE REVIEW

**Comment by pjcozzi** [comment]
Looks good, please update this part of README.md: https://github.com/AnalyticalGraphicsInc/gltf-pipeline#building-for-cesium-integration

**Comment by pjcozzi** [comment]
> Note: The nomenclature (simple/simplify) maybe naive. So if there is a technical term for it, then I can change it to that.

Used `combined`/`combine`

File: lib/getStatistics.js [hunk 1]
```diff
@@ -58,6 +57,28 @@ Statistics.prototype.toString = function() {
...
 };
 
+function isDataUri(uri) {
+    //Return false if the uri is undefined
+    if (!defined(uri)) {
+        return false;
+    }
+
+    //Return true if the uri is a data uri
+    return /^data\:/i.test(uri);
```
Comment (lilleyse) [inline lib/getStatistics.js]: Add one space between `//` and the comment.

File: lib/getStatistics.js [hunk 2]
```diff
@@ -97,11 +118,78 @@ function getNumberOfProperties(object) {
     return Object.keys(object).length;
 }
 
+function getDrawCallStatisticsForNode(gltf, nodeId) {
```
Comment (lilleyse) [inline lib/getStatistics.js]: `getDrawCallStatisticsForNode` and  `getDrawCallStatistics` are similar enough that they should share a helper function that gets the statistics for an array of primitives.

File: lib/getStatistics.js [hunk 3]
```diff
@@ -144,18 +232,29 @@ function getDrawCallStatistics(gltf) {
...
  */
-function getStatistics(gltf) {
+function getStatistics(gltf, nodeId) {
     if (!defined(gltf)) {
         throw new DeveloperError('gltf must be defined');
     }
 
+    var statistics = new Statistics();
+
+    if(defined(nodeId)) {   // Only do draw call statistics
```
Comment (lilleyse) [inline lib/getStatistics.js]: `if(` -> `if (`

File: README.md [hunk 4]
```diff
@@ -77,6 +77,15 @@ npm run build-cesium
 
 This will output a portion of the gltf-pipeline code into the `dist/cesium` folder, reformatted into AMD style for use with RequireJS and Cesium in the browser.
 
+### Building for Other integration
```
Comment (lilleyse) [inline README.md]: Change Other to other.

**Comment by shehzan10** [comment]
@lilleyse updated.

**Comment by lilleyse** [comment]
Looks good!

**Pull request merged**